runner/src/server/plugins/engine/views/components/clientsidefileuploadfield.html

```diff
@@ -409,16 +409,16 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
         }
 
         myDropzone.on('error', function (file, error) {
-            if (error === "You can't upload files of this type.") {
+            if (error === `{{ i18nGetTranslation("validation.fileUpload.fileUploadTypeError") }}`) {
```

Comment:
Having branching error condition logic based on strings that are generated by a translation process feels quite delicate! I guess one way of thinking about it is that there's a few things that could change here without changing this code that could break it?

I've not read the code in detail but I wonder if its possible to use constants for error types/ names that are used for the logic and then mapping out to a translated message seperately.